ports/espressif/peripherals/esp32p4/pins.c

```diff
@@ -0,0 +1,53 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2021 microDev
+//
+// SPDX-License-Identifier: MIT
+
+#include "peripherals/pins.h"
+
+const mcu_pin_obj_t pin_GPIO0 = PIN(0, NO_ADC, NO_ADC_CHANNEL, NO_TOUCH_CHANNEL);
+const mcu_pin_obj_t pin_GPIO1 = PIN(1, ADC_UNIT_1, ADC_CHANNEL_0, TOUCH_PAD_NUM1);
+const mcu_pin_obj_t pin_GPIO2 = PIN(2, ADC_UNIT_1, ADC_CHANNEL_1, TOUCH_PAD_NUM2);
```

Comment:
This seems to differ from the datasheet
![image](https://github.com/user-attachments/assets/1ce80024-95bb-47c0-a776-5a713ce510f2)

Reply:
Updated. Thanks for checking!